src/components/data/listing/Listing.js

```diff
@@ -18,7 +18,7 @@ import constants from "../../../constants";
 import DEPagination from "components/utils/DEPagination";
 import ResourceTypes from "components/models/ResourceTypes";
 import isQueryLoading from "components/utils/isQueryLoading";
-import URLImportDialog from "components/URLImportDialog";
+import URLImportDialog from "components/urlImport";
```

Comment:
Updated folder name for consistency (lowercase)

Reply:
Is this consistent? I thought that we used initial uppercase for components (like `URLImportDialog`) and lowercase for non-component stuff like `isQueryLoading`. I'm not as used to this code, of course, but I'd expect it to be the former name.

Reply:
I just renamed the folder not the the file `index.js` that contains the `URLImportDialog`. (folder `URLImportDialog` was renamed to `urlImport `). Folders containing the components should use all lowercase.